src/internal.c

```diff
@@ -3544,6 +3605,19 @@ static int DoGlobalRequest(WOLFSSH* ssh,
         ret = GetBoolean(&wantReply, buf, len, &begin);
     }
 
+#ifdef WOLFSSH_FWD
+    if (ret == WS_SUCCESS) {
+        if (WSTRNCMP(name, "tcpip-forward", nameSz) == 0) {
+            fprintf(stderr, "tcpip-forward requested\n");
```

Comment:
frpintf and fflush should be mapped from a macro for portability

Reply:
They shouldn't be used here at all. That should be a WLOG().

Reply:
I deleted that block because while I was checking for the string I wasn't doing anything with it.